wemake_python_styleguide/visitors/ast/imports.py

```diff
@@ -27,7 +28,7 @@
 ErrorCallback = Callable[[BaseViolation], None]  # TODO: alias and move
 
 
-@final
+@final  # noqa: WPS214
```

Comment:
Can we do something with this violation here?

Can we refactor this class to be more simple? Or just split it?

Reply:
In the latest [commit](https://github.com/wemake-services/wemake-python-styleguide/pull/1118/commits/f8dc24e45cc18d0d810797c0b9052e2d0c67698d) I splitted this validator into 3 classes:

+ `ast.Import` validator
+ `ast.ImportFrom` validator
+ base class for both above mentioned validators

If you have any ideas how to make it clearer please let me know.